sdk/core/Azure.Core/tests/HttpMessageTests.cs

```diff
@@ -241,6 +241,72 @@ public void AppliesHandlerBeforeResponseClassifier()
             Assert.IsTrue(message.ResponseClassifier.IsErrorResponse(message));
         }
 
+        [Test]
+        public void AppliesResponseClassifier()
```

Comment:
At first glance it wasn't clear to me what should have been applied here. Would something like this more accurately describe this test?

```suggestion
        public void ApplyingEmptyResponseClassifierDoesNotChangeClassification()
```

Reply:
Ah, but the response classifier is not empty! :)  It is a special HeadResponseClassifier type.

But point taken, happy to write a more descriptive name for the test.